bsp/boards/iot-lab_A8-M3/board.c

```diff
@@ -96,8 +101,48 @@ void board_init(void){
     debugpins_init();
     //enable nvic for the radio
     NVIC_radio();
+    
+    board_init_slot_vars();

```

Comment:
don't call another function, fill in the structre here.

Reply:
What about having an approach like https://github.com/openwsn-berkeley/openwsn-fw/pull/504#discussion_r410117118? You would need fewer lines of code modifications in every `board_info.h`, and could have a default for all boards which parameters have not been measured, you could initialize the structure outside of `bsp/` as well.

Reply:
@twatteyne not sure I understand, which structure do you mean? 
@fjmolinas  well, the general idea that slot selection will be done at runtime and the selection is done at MAC level (and the MAC level will have the default selection) but we can discuss more in the meeting to make sure I understood you correctly! Thanks a lot for your feedback!

Reply:
@minarady1 I PR'd what I meant to your branch https://github.com/minarady1/openwsn-fw/pull/1. Did it only for 1 board.